awscli/customizations/cloudtrail.py

```diff
@@ -282,8 +284,9 @@ def setup_new_topic(self, topic, policy_url=None):
         if policy_url:
             policy = requests.get(policy_url).text
         else:
-            data = self.s3.GetObject(bucket='awscloudtrail',
-                                     key=SNS_POLICY_TEMPLATE)
+            data = self.s3.GetObject(
```

Comment:
Same thing here (better error messages).  Might be worth refactoring now too since this looks like dupe code from line 209.